pkg/controllers/payload/payload_config_controller.go

```diff
@@ -88,8 +90,8 @@ func NewPayloadConfigController(kubeInformersForTargetNamespace informers.Shared
 	).ResyncEvery(wait.Jitter(time.Minute, 1.0)).WithSync(c.sync).ToController("PayloadConfig", recorder.WithComponentSuffix("payload-config-controller"))
 }
 
-func (c *payloadConfigController) getAuthConfig(ctx context.Context) (*operatorv1.Authentication, []operatorv1.OperatorCondition) {
-	operatorConfig, err := c.auth.Authentications().Get(ctx, "cluster", metav1.GetOptions{})
+func (c *payloadConfigController) getAuthConfig(ctx context.Context) (*operatorv1.OperatorSpec, []operatorv1.OperatorCondition) {
```

Comment:
`ctx` is unused.

```suggestion
func (c *payloadConfigController) getAuthConfig() (*operatorv1.OperatorSpec, []operatorv1.OperatorCondition) {
```

Reply:
> `ctx` is unused.

Let's plumb it through everywhere for future usage.  It's a good practice.